docs: updates Azure VPC peering service principal (#782)
kyma-project/cloud-manager: docs/user/resources/04-40-10-azure-vpc-peering.md

Kyma and the remote Azure Virtual Private Cloud (VPC) network. Virtual network peering is only possible within the networks
of the same cloud provider.

Once an `AzureVpcPeering` CR is created and reconciled, the Cloud Manager controller creates a VPC peering connection in

> the VPC network of the Kyma cluster in the underlying cloud provider subscription and accepts VPC peering connection in 

Reviewer: ```suggestion
the VPC network of the Kyma cluster in the underlying cloud provider subscription, and accepts a VPC peering connection in 
```
I think "connection" needs the "a", but let me know if I misunderstand the sentence

the remote cloud provider subscription.

### Authorization

> Cloud Manager must be authorized in the remote cloud provider subscription to accept VPC peering connection.

Reviewer: ```suggestion
Cloud Manager must be authorized in the remote cloud provider subscription to accept a VPC peering connection.
```

Assign the following IAM roles to the Cloud Manager service principal `kyma-cloud-manager-peering-ENV` in the remote subscription: 
* Classic Network Contributor
* Network Contributor

**ENV** corresponds to **dev**, **stage**, or **prod**.

### Deleting `AzureVpcPeering`

> Kyma's underlying cloud provider VPC peering connection is deleted as a part of AzureVpcPeering deletion. The remote VPC

Reviewer: ```suggestion
Kyma's underlying cloud provider VPC peering connection is deleted as a part of the AzureVpcPeering deletion. The remote VPC
```

> peering connection is left hanging, and must be deleted manually.

Reviewer: ```suggestion
peering connection is left hanging, and you must delete it manually.
```
